src/System.Web.Http/Validation/Validators/DataAnnotationsModelValidator.cs

```diff
@@ -33,11 +33,11 @@ public override IEnumerable<ModelValidationResult> Validate(ModelMetadata metada
             // Per the WCF RIA Services team, instance can never be null (if you have
             // no parent, you pass yourself for the "instance" parameter).
             
-            string memberName = metadata.GetDisplayName();
+            var memberName = metadata.PropertyName;
```

Comment:
For a while, aspnet/MVC's version of this file used `metadata.PropertyName ?? metadata.ModelType.Name`. But, though that's more like what `GetDisplayName()` did, it's also incorrect. We fixed it to look like the above in aspnet/Mvc@aecfe778ee.

Reply:
I'm on the fence about this particular expression. In favour of a change: We recognized a problem and fixed that problem in ASP.NET Core MVC. Against: MVC 5.2.4 still uses `metadata.PropertyName ?? metadata.ModelType.Name` i.e. the consistency across ASP.NET argument. And, going this way makes it a larger change i.e. `MemberName` will now ignore both `[DisplayName("some name")]` and the type name for non-properties.